src/sentry/web/client_config.py

```diff
@@ -224,6 +224,11 @@ def enabled_features(self) -> Iterable[str]:
             yield "relocation:enabled"
         if features.has("system:multi-region"):
             yield "system:multi-region"
+        # TODO @athena: remove this feature flag after development is done
+        # this is a temporary hack to be able to used flagpole in a case where there's no organization
+        # availble on the frontend
+        if self.last_org and features.has("organizations:scoped-partner-oauth", actor=self.user):
```

Comment:
You'll need an organization as well to have flagpole actually do a feature flag check.